lib/puppet/indirector/trusted_ext/plain.rb

```diff
@@ -0,0 +1,11 @@
+require 'puppet/trusted_ext'
+require 'puppet/indirector/plain'
+
+class Puppet::TrustedExt::Plain < Puppet::Indirector::Plain
+  desc "Always return an empty Hash. This is a placeholder to make the default configuration a noop."
```

Comment:
Can you default the terminus to `:none` instead? There's already a `Puppet::Indirector::None` terminus whose find method returns nil.

Reply:
done.